src/mm-tabs/index.html

```diff
@@ -0,0 +1,112 @@
+<!DOCTYPE html>
+<html>
+  <head>
+		<script language="javascript" src="../../bower_components/webcomponentsjs/webcomponents-lite.js"></script>
+		<link rel="import" href="../mm-icon/mm-icon.html">
+		<link rel="import" href="../mm-tab/mm-tab.html">
+		<link rel="import" href="mm-tabs.html">
+		<style type="text/css">
+			body, html {
+				height: 100%;
+				min-height: 100%;
+			}
+
+			body {  
+				margin:0;
+				padding:0;
+				background: #eee;
+			}
+
+			body /deep/ .col,
+			.col {
+				-webkit-box-sizing: border-box;
+				-moz-box-sizing: border-box;
+				box-sizing: border-box;
+				display: block;
+				float: left;
+				padding: 0 20px;
+				/*font-size: 0;*/
+			}
+
+			.col.c0 {
+				width:400px;
+			}
+
+			.col.c1 {
+				width:300px;
+			}
+
+			.col.c2 {
+				width:200px;
+			}
+
+			.col.c3 {
+				width:500px;
+			}
+
+			body /deep/ .col.c4,
+			.col.c4 {
+				width:50%;
+			}
+
+			.col:after {
+				visibility: hidden;
+				display: block;
+				font-size: 0;
+				content: " ";
+				clear: both;
+				height: 0;
+			}
+
+			body /deep/ p,
+			p {
+				font-family: sans-serif;
+				color: #333;
+				font-size: 12px;
+				line-height: 18px;
+				margin: 20px 0;
+			}
+
+			.bold { font-weight: bold; }
+
+			hr {
+				border: 0;
+				display: block;
+					border-bottom: 1px solid #fff;
+					border-top: 1px solid #ccc;
+					height: 0px;
+					margin-top: 20px;
+			}
+
+			.vr {
+				width:0px;
+				height: 100%;
+				border-right: 1px solid #fff;
+				border-left: 1px solid #ccc;
+				display: block;
+				float: left;
+			}
+		</style>
+  </head>
+  <body>
+  	<div style="padding-right: 20px;">
+		<mm-tabs tab-bar-offset="4" id="tabSection" tab-bar-position="left">
+			<mm-tab id="ext" tab-label="External URL with tabs" icon="apps" url="tab-content.html"></mm-tab>
+			<mm-tab tab-label="Content in Light DOM" icon="pixels">
+				<div class="col c4">
+					<p>The ostensible reason why Ahab did not go on board of the whaler we had spoken was this: the wind and sea betokened storms. But even had this not been the case, he would not after all, perhaps, have boarded her&mdash;judging by his subsequent conduct on similar occasions&mdash;if so it had been that, by the process of hailing, he had obtained a negative answer to the question he put. For, as it eventually turned out, he cared not to consort, even for five minutes, with any stranger captain, except he could contribute some of that information he so absorbingly sought. But all this might remain inadequately estimated, were not something said here of the peculiar usages of whaling-vessels when meeting each other in foreign seas, and especially on a common cruising-ground.</p>
+				</div>
+			</mm-tab>
+			<mm-tab id="jsContent" tab-label="JS Content Load"></mm-tab>
+		</mm-tabs>
+	</div>
+
+	<script>
+	document.addEventListener('WebComponentsReady', function() {
+		document.querySelector('#jsContent').loadExternal('tab-content-alt.html', function() {
```

Comment:
can we also make this automatic when the content is specified as a tab attribute ie

```html
<mm-tab tab-label="Load some stuff" id="jsContent" content="some-import.html"></mm-tab>
```